rfps/near-and-filecoin.md

```diff
@@ -0,0 +1,62 @@
+# NEAR x Filecoin
+
+This is a Request for Proposals for projects interested in exploring new opportunities at the intersection of NEAR and Filecoin.
+
+[NEAR](https://near.org/) is a decentralized application platform built to bridge the users of today’s Internet to the blockchain-based web of the future. NEAR is a PoS Layer 1 blockchain platform, built with UX and DevX in mind. NEAR’s novel sharding mechanism, Nightshade, parallelizes computation on chain and automatically increases capacity when needed, resulting in theoretically infinite scalability.
+
+[Filecoin](https://filecoin.io/) is a decentralized peer-to-peer storage network, allowing anyone to store and validate large-scale data as part of a global, 8-exabyte (and growing) network. Under the hood, all Filecoin nodes use the InterPlanetary File System (IPFS), a content-addressed protocol for storing and sharing data.
+
+In July 2021, the [Filecoin-NEAR bridge](https://blog.textile.io/native-filecoin-storage-for-blockchains/) developed by Textile was launched, taking the first step to provide simple, permissionless storage for smart contracts on NEAR. 
+
+This RFP is looking to deepen the integrations between NEAR and Filecoin, building on the successful launch of the bridge.
+
+NEAR and Filecoin are jointly offering up to **USD $300,000** in grant funding for eligible projects.
+
+Below are a few examples of possible new applications, but all proposals combining NEAR and Filecoin are encouraged to apply:
+
+## 1. **Decentralized Storage SDK + Demonstration app**
```

Comment:
Todo: there's a thread fleshing out these ideas a bit more in GDoc, reminder to incorporate those updates once resolved.